scripts/build-for-upload.sh

```diff
@@ -3,34 +3,38 @@
 ## exit if an error occurs or on unset variables
 set -eu -o pipefail
 
-declare -r SIMLAR_KEYSTORE=${SIMLAR_KEYSTORE:-""}
-declare -r KEYSTORE=${1:-"${SIMLAR_KEYSTORE}"}
+declare -r  SIMLAR_KEYSTORE=${SIMLAR_KEYSTORE:-""}
+declare -rx KEYSTORE_FILE=${1:-"${SIMLAR_KEYSTORE}"}
 
-declare -r GRADLEW="$(dirname $(readlink -f $0))/../gradlew"
+declare -r  PROJECT_DIR="$(dirname $(readlink -f $0))/.."
+declare -r  GRADLEW="${PROJECT_DIR}/gradlew"
 
-if [ -z "${KEYSTORE}" ] ; then
+if [ -z "${KEYSTORE_FILE}" ] ; then
 	echo "Please set give parameter keystore, e.g.:"
 	echo "  $0 ~/dev/android/simlar-release-key.keystore"
 	echo "or set the environment variable KEYSTORE, e.g.:"
 	echo "  export SIMLAR_KEYSTORE=~/dev/android/simlar-release-key.keystore ; $0"
 	exit
 fi
 
-echo "using keystore ${KEYSTORE}"
+echo "using keystore ${KEYSTORE_FILE}"
+echo "enter password its password:"
```

Comment:
`its password`? Or `, the password is "password"`?

Reply:
Ups. Thanks again!